packages/server/src/launcher.ts

```diff
@@ -83,6 +83,9 @@ export class Launcher {
     if (!this.shuttingDown) {
       this.shuttingDown = true
 
+      this.logger.info('Server gracefully closing down...')
+
```

Comment:
Only print this if we're not spinned from botpress

`if (!yn(process.env.SPINNED)) {`

Also I think a message `Shutdown complete` or something after would be useful.